package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "timed",
-  "version": "2.1.3",
+  "version": "3.0.0-rc12",
```

Pretty sure it's not the 12th release candidate :wink: Also, the rc version is delimited by a period.
```suggestion
  "version": "3.0.0-rc.1",
```

---

Well, we are hitting the twelfth pre-release... since this is the last tag [v3.0.0-rc.11](https://github.com/adfinis/timed-frontend/releases/tag/v3.0.0-rc11)

---

... `rc` not used in an exemplary manner

---

Ah, I see. The package version was not being updated..